validate-seqtools/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:18.04
+
+LABEL org.opencontainers.image.source https://github.com/icgc-argo/argo-data-submission
+LABEL org.opencontainers.image.authors Edmund Su (edmund.su@oicr.on.ca)
+LABEL org.opencontainers.image.title ICGC ARGO Data download image
```

Comment:
ICGC ARGO Data Submission Validation image?

Reply:
Edited to `ICGC ARGO Data Submission Validation image`
https://github.com/icgc-argo/argo-data-submission/blob/2ed0573fbe4a1be55ec51b67a3e56d2359b139c6/validate-seqtools/Dockerfile#L5